Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -303,18 +303,22 @@ import Foundation
     @_documentation(visibility: private)
     public func sendAnalyticsEvent(
         _ eventName: String,
+        buttonRank: Int? = nil,
```

Comment:
Curious why this is optional? Don't all buttons have rank/order?

Reply:
great question - not sure why it's optional either but this is what was passed down to us from product, however, now we're no longer using `buttonRank` anymore as it's been updated to `paymentMethodsDisplayed` now